news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2017 Signups now Open"
+permalink: no idea what to do with these fields
+date: 2017-07-xx xx:xx:xx +0000
+tumblr_url: no idea what to do with these fields
+---
+
+Have you ever thought: “Wow, I’m bad at mapping”? Does the quote “So let’s be honest, you suck at mapping” (pishifat, 2016)  strike a personal chord with you? Then the Community Mentorship Program is the place for you!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/ZXakpTkXEC0" frameborder="0" allowfullscreen></iframe>
+
+[seems to need a banner of sorts?]
```

Comment:
banner goes before iframe content, you'll need to make one or have one made for this. check previous newsposts for the particular dimensions, we have a "something" newsposts template using scorewatch/osuweekly motifs (but non coloured) we can use if you haven't got anything